### DISCUSSION TITLE
[Bug]: Runtime error

### DISCUSSION THREAD

**Discussion by chokiproai** [danny-avila/LibreChat#1066] [Troubleshooting] [answered]
### What happened?

![image](https://github.com/danny-avila/LibreChat/assets/128912789/97b4639f-87a4-44f7-b323-b25cea9e9d82)

### Steps to Reproduce

error

### Relevant log output

```shell
> LibreChat@0.5.9 backend
> cross-env NODE_ENV=production node api/server/index.js
Clearing sync timeouts before exiting...
There was an uncaught error:
MeiliSearchCommunicationError: Service Unavailable
at /app/node_modules/meilisearch/dist/bundles/meilisearch.cjs.js:182:27
at step (/app/node_modules/meilisearch/dist/bundles/meilisearch.cjs.js:103:23)
at Object.throw (/app/node_modules/meilisearch/dist/bundles/meilisearch.cjs.js:84:53)
at rejected (/app/node_modules/meilisearch/dist/bundles/meilisearch.cjs.js:75:65) at process.processTicksAndRejections (node: internal/process/task_queues:95:5) {
errno: undefined,
code: undefined
}
npm notice
npm notice New major version of npm available! 9.6.3 10.2.0
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.2.0> npm notice Run npm install -g npm@10.2.0 to update!
npm notice
```

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/128912789/49bfc645-f5ea-4f10-9930-1835f8db9901)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
Hi, thanks for checking out the project.

Sadly, I can't really help here without more context. Moved to troubleshooting because meilisearch is working when setup properly, and the exact runtime error is irreproducible.

How did you install the project? Are you trying to use meilisearch? Did you follow any of the guides here? 
https://docs.librechat.ai/install/docker_compose_install.html

If so please link the one you used.

**Comment by chokiproai** [comment]
Thank you it worked, I use huggingface and run docker
